src/GameLogic/ItemPriceCalculator.cs

```diff
@@ -18,10 +18,15 @@ namespace MUnique.OpenMU.GameLogic;
 public class ItemPriceCalculator
 {
     private const short ForceWaveSkillId = 66;
-    private const long MaximumPrice = 3000000000;
+    private const short ExplosionSkillId = 223;
+    private const short RequiemSkillId = 224;
+    private const short PollutionSkillId = 225;
+    private const long MaximumPrice = 3_000_000_000;
     private const float DestroyedPetPenalty = 2.0f;
     private const float DestroyedItemPenalty = 1.4f;
 
+    private static readonly List<short> WorthlessSkills = [ForceWaveSkillId, ExplosionSkillId, RequiemSkillId, PollutionSkillId];
```

Comment:
There may be more "worthless" skills. This is what I found. 
![Screenshot from 2024-12-05 20-40-54](https://github.com/user-attachments/assets/6ea0735f-4c86-4901-9c9d-207cf7470c2a) ![Screenshot from 2024-12-05 20-26-12](https://github.com/user-attachments/assets/38fef33a-2828-4c2a-a82c-7ffd02277668)